src/Sarif.Converters.UnitTests/TSLintLoaderTests.cs

```diff
@@ -85,7 +85,7 @@ public void TSLintLoader_ReadLog_ProducesExpectedLog()
                 }
             };
 
-            TSLintLoader loader = new TSLintLoader();
+            TSLintLogReader loader = new TSLintLogReader();
```

Comment:
Rename `loader` => `logReader`, hear and every place a variable of that name actually refers to a `TSLintLogReader`.